datalad/support/github_.py

```diff
@@ -270,37 +300,50 @@ def _make_github_repo(github_login, entity, reponame, existing,
             reponame)
 
     if repo is not None:
+        res = dict(
+            url=access_url,
+            prexisted=True,
+        )
         if existing in ('skip', 'reconfigure'):
```

Comment:
Hmm, not new to this PR, but a note to myself that the help output should probably be updated to note that 'reconfigure' really just behaves in the same way as 'skip'.

Reply:
Yeah, that bothered me too. I expected to find a difference, but if there is one, it can only be in the `siblings()` call, and here it is just used for a non-intuitive pass-through.

Reply:
hm, indeed.  FWIW: I thought that may be some outside logic behaves differently between the two, but found no such spot.